### ISSUE TITLE
Sorting orders by users in backend creates wrong sql query

### ISSUE THREAD

**Issue by Dukecz** [Sylius/Sylius#2313]
In backend, when I try to sort orders by user (administration/orders/?sorting%5Buser%5D=asc) exception is thrown thanks to missing column in select part of sql:

>An exception has been thrown during the rendering of a template ("An exception occurred while executing 'SELECT DISTINCT s0_.id AS id0, s0_. AS _1 FROM sylius_order s0_ WHERE (s0_.completed_at IS NOT NULL) AND (s0_.deleted_at IS NULL) ORDER BY s0_.user_id ASC LIMIT 20 OFFSET 0':

>SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'AS _1 FROM sylius_order s0_ WHERE (s0_.completed_at IS NOT NULL) AND (s0_.delete' at line 1") in SyliusWebBundle:Backend/Order:index.html.twig at line 30.

While the correct sql should be (s0_. => s0_.user_id)
```mysql
SELECT DISTINCT s0_.id AS id0, s0_.user_id AS _1 FROM sylius_order s0_ WHERE (s0_.completed_at IS NOT NULL) AND (s0_.deleted_at IS NULL) ORDER BY s0_.user_id ASC LIMIT 20 OFFSET 0
```
Sorting using other columns work just fine.

**Comment by Dukecz** [comment]
Thanks @BartoszSiejka

**Comment by stloyd** [comment]
@pjedrzejewski @aRn0D This was fixed by #2345...
